applicationLibrary/molecularDynamics/molecularDynamicsLibrary/LJFunctor.h

```diff
@@ -684,6 +737,100 @@ class LJFunctor
     return _virialSum[0] + _virialSum[1] + _virialSum[2];
   }
 
+  /**
+   * Gets the number of useful FLOPs.
+   *
+   * For the distance squared calculation, this is:
+   * - Displacement: 3
+   * - DistanceSquared in each dimension: 3
+   * - DistanceSquared Total: 2
+   * - Total: 8
+   *
+   * For the force kernel, this is:
+   * - inverse distance squared: 1 (assume division is 1 FLOP)
+   * - lj2: 1
+   * - lj6: 2
+   * - lj12: 1
+   * - lj12m6: 1
+   * - scalar factor: 3
+   * - force: 3
+   * - accumulate force on mol i: 3
+   * - accumulate force on mol j if n3: 3
+   * - Total: 15 without n3, 18 with n3
+   *
+   * For the globals calculation, this is:
+   * - virial: 3
+   * - potential: 1, or 2 with shift
+   * - accumulation: 4
+   * - multiplications by 0.5 and second accumulations are not treated as useful FLOPs, as these are just workarounds
+   *   to avoid accumulating globals on owned/halo borders twice
+   * - Total: 8, or 9 with shift
+   *
+   * Caveats:
+   *
+   * This function is supposed to return useful FLOPs, e.g. without counting masked vector instructions or
+   * workarounds for accumulating globals on owned/halo borders twice. You could argue that, strictly speaking, we
+   * redundantly calculate forces and globals twice in the newton3 case. This function does not treat such "redundant"
+   * calculations as useless.
+   *
+   * Similarly, this function does not treat halo-halo interactions as redundant useless calculations.
+   *
+   * @return number of FLOPs since initTraversal() is called.
+   */
+  size_t getNumFLOPs() {
```

Comment:
```suggestion
  size_t getNumFLOPs() override {
```
Also missing for all other instantiations of this function.

Reply:
Done in 7fa81126